module/VuFind/src/VuFind/Log/LoggerFactory.php

```diff
@@ -58,173 +75,147 @@ class LoggerFactory implements FactoryInterface
     use EmailSettingsTrait;
 
     /**
-     * Configure database writers.
-     *
-     * @param Logger             $logger    Logger object
-     * @param ContainerInterface $container Service manager
-     * @param string             $config    Configuration
+     * Get a standard LineFormatter for file logging.
      *
-     * @return void
+     * @return LineFormatter
      */
-    protected function addDbWriters(
-        Logger $logger,
-        ContainerInterface $container,
-        $config
-    ) {
-        $parts = explode(':', $config);
-        $table_name = $parts[0];
-        $error_types = $parts[1] ?? '';
-
-        $columnMapping = [
-            'priority' => 'priority',
-            'message' => 'message',
-            'logtime' => 'timestamp',
-            'ident' => 'ident',
-        ];
-
-        // Make Writers
-        $filters = explode(',', $error_types);
-        $writer = new Writer\Db(
-            $container->get(\VuFind\Db\Connection::class),
-            $table_name,
-            $columnMapping
+    protected function getStandardFileFormatter(): LineFormatter
+    {
+        return new LineFormatter(
+            "[%datetime%] %channel%.%level_name%: %message% %context% %extra%\n",
+            'Y-m-d H:i:s',
+            true,
+            true
         );
-        $this->addWriters($logger, $writer, $filters);
     }
 
     /**
-     * Configure email writers.
+     * Configure Database handler.
      *
-     * @param Logger             $logger    Logger object
-     * @param ContainerInterface $container Service manager
+     * @param MonologLogger      $logger    The Monolog logger instance to add handlers to.
      * @param Config             $config    Configuration
+     * @param ContainerInterface $container Service manager
      *
      * @return void
      */
-    protected function addEmailWriters(
-        Logger $logger,
-        ContainerInterface $container,
-        Config $config
-    ) {
-        // Set up the logger's mailer to behave consistently with VuFind's
-        // general mailer:
-        $parts = explode(':', $config->Logging->email);
-        $email = $parts[0];
+    protected function addDbHandler(MonologLogger $logger, Config $config, ContainerInterface $container)
+    {
+        $parts = explode(':', $config->Logging->database);
+        $table_name = $parts[0];
         $error_types = $parts[1] ?? '';
-
-        // Make Writers
         $filters = explode(',', $error_types);
-        $writer = new Writer\Mail(
-            $container->get(\VuFind\Mailer\Mailer::class),
-            $this->getEmailSenderAddress($config),
-            $email,
-            'VuFind Log Message'
-        );
-        $this->addWriters($logger, $writer, $filters);
+
+        $connection = $container->get(\VuFind\Db\Connection::class);
+        $baseDatabaseHandler = new \VuFind\Log\Handler\DatabaseHandler($connection, $table_name);
```

Comment:
You're using a fully qualified class name here, but some of the other handlers have use statements. It's probably best to be consistent and use the same style for referencing all of the handlers. If you want a compromise between a use statement and a FQCN, you could also use a relative name like `Handler\DatabaseHandler` since this file is in the `VuFind\Log` namespace already.